### PR TITLE
Cart component

### PR DESCRIPTION
**Pull request by raghav6202**
Please review my cart component.The components are in a single page I'll be adding the links in the next PR.

### CODE REVIEW

File: src/context/wishlist.context.jsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+const wishlistReducerFn = (state, action) => {
+  switch (action.payload) {
+    case "add_to_wishlist":
+      return {
+        ...state,
+        wishlist: [...state.wishlist, { ...action.payload, wishes: 1 }]
+      };
+
+    case "remove_from_wishlist":
```
Comment (saketkum11) [inline src/context/wishlist.context.jsx:13]: using Uppercase letter Would be recommended

File: src/comps/cart-items/cart-items.jsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+                cartDispatch({ type: "decrement", payload: cartInfo })
+              }
+            >
+                <span className="quantity-icon"><HiMinus /></span>
+    
+            </button>
+            {cartInfo.quantity}
+            <button
+              className="quantity-btn"
+              onClick={() => cartDispatch({ type: "increment", payload: cartInfo })}
```
Comment (saketkum11) [inline src/comps/cart-items/cart-items.jsx:40]: use Type in Uppercase

**Review by saketkum11 — APPROVED** [review]
- clean code
- short review code great.
- easy to understand

File: src/context/wishlist.context.jsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
+import { createContext, useContext, useReducer } from "react";
+
+const ContextOfWishlist = createContext(null);
+
+const wishlistReducerFn = (state, action) => {
+  switch (action.payload) {
+    case "add_to_wishlist":
```
Comment (arulk8) [inline src/context/wishlist.context.jsx:7]: this actions type can be placed inside a constants file and can be used any place we want

**Review by arulk8 — APPROVED** [review]
The action ca

**Pull request merged**